generic.resoscript

```diff
@@ -0,0 +1,815 @@
+    <Parameter Name="UniqueIDValue" Value="REQUIRED: YOUR UniqueIDValue GOES HERE" />
+
+    <!-- For Top, Skip, and Sort Testing -->
+    <Parameter Name="TopCount" Value="5" />
+    <Parameter Name="SortCount" Value="20" />
+
+    <!-- Numeric Field for eq, ne, gt, ge, lt, le testing -->
+    <!-- Type="Edm.Int16" or Type="Edm.Int32" or Type="Edm.Int64" for non-lookup fields -->
+    <Parameter Name="FilterNumericField" Value="BedroomsTotal" />
+    <Parameter Name="FilterNumericValueLow" Value="9" />
+    <Parameter Name="FilterNumericValueHigh" Value="15" />
+    <Parameter Name="FilterNumericNotFound" Value="-1" />
+
+    <!-- Numeric Field for "not" testing -->
+    <Parameter Name="FilterNotField" Value="BedroomsTotal" />
+    <Parameter Name="FilterNotValue" Value="-1" />
+
+    <!-- Enumerated Field for "has" testing -->
+    <Parameter Name="FilterHasField" Value="PropertyType" />
+    <Parameter Name="FilterHasValue" Value="PropertyEnums.PropertyType'Rental'" />
+
+    <!-- Enumerated Field for SingleValue/Multi-value testing -->
+    <Parameter Name="SingleValueLookupField" Value="PropertyType" />
+    <Parameter Name="SingleValueLookupValue" Value="PropertyEnums.PropertyType'Rental'" />
+    <Parameter Name="MultipleValueLookupField" Value="Appliances" />
+    <Parameter Name="MultipleValueLookupValue1" Value="PropertyEnums.Appliances'Refrigerator'"/>
+    <Parameter Name="MultipleValueLookupValue2" Value="PropertyEnums.Appliances'Stacked'" />
+
+    <!-- For Expand Testing -->
+    <Parameter Name="ExpandField" Value="ListAgent" />
+
+    <!-- For Geo-spatial Testing -->
+    <Parameter Name="GeospatialLatitudeField" Value="Latitude" />
+    <Parameter Name="GeospatialLongitudeField" Value="Longitude" />
+    <Parameter Name="GeospatialField" Value="Coordinates" />
+    <Parameter Name="GeospatialValue" Value="REQUIRED: YOUR COORDINATES GO HERE in 'Longitude Latitude' format" /> <!-- "Longitude Latitude" -->
+    <Parameter Name="GeospatialDistanceValue" Value="100" />
+
+    <!--String Fields for testing -->
+    <!-- Type="Edm.String" -->
+    <Parameter Name="ContainsField" Value="StreetName" />
+    <Parameter Name="ContainsValue" Value="M" />
+    <Parameter Name="EndsWithField" Value="StreetName" />
+    <Parameter Name="EndsWithValue" Value="Rd" />
+    <Parameter Name="StartsWithField" Value="StreetName" />
+    <Parameter Name="StartsWithValue" Value="M" />
+    <Parameter Name="ToLowerField" Value="StreetName" />
+    <Parameter Name="ToLowerValue" Value="main" />
+    <Parameter Name="ToUpperField" Value="StreetName" />
+    <Parameter Name="ToUpperValue" Value="MAIN" />
+
+    <!-- REQUIRED: Date Fields for testing -->
+    <Parameter Name="DateField" Value="ModificationTimestamp" />
+    <Parameter Name="DateTimeValue" Value="2018-12-31T23:55:55-09:00" />
+    <Parameter Name="DateValue" Value="2018-12-31" />
+    <Parameter Name="YearValue" Value="2018" / >

```

Comment:
Because there is a space here there seems to be an exception that gets thrown when attempting to use the `--runRESOScript` command. 

```
[Fatal Error] :122:47: Element type "Parameter" must be followed by either attribute specifications, ">" or "/>".
org.xml.sax.SAXParseException; lineNumber: 122; columnNumber: 47; Element type "Parameter" must be followed by either attribute specifications, ">" or "/>".
        at com.sun.org.apache.xerces.internal.parsers.DOMParser.parse(DOMParser.java:257)
        at com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderImpl.parse(DocumentBuilderImpl.java:339)
        at javax.xml.parsers.DocumentBuilder.parse(DocumentBuilder.java:121)
        at org.reso.resoscript.ClientSettings.loadFromRESOScript(ClientSettings.java:66)
        at org.reso.resoscript.Settings.loadFromRESOScript(Settings.java:41)
        at org.reso.commander.Main.main(Main.java:81)
```

```suggestion
    <Parameter Name="YearValue" Value="2018" />
```